doc/changelog/07-vernac-commands-and-options/15366-master+Fail-not-catching-anomaly.rst

```diff
@@ -0,0 +1,5 @@
+- **Changed:**
+  :cmd:`Fail` caught anomalies from Coq version 8.11; this has been
+       reverted to catching only proper user errors
+  (`#15366 <https://github.com/coq/coq/pull/15366>`_,
+  by Hugo Herbelin).
```

Comment:
```suggestion
  :cmd:`Fail` no longer catches anomalies. A behavior present since
       Coq version 8.11. :cmd:`Fail` now only catches user errors
  (`#15366 <https://github.com/coq/coq/pull/15366>`_,
  by Hugo Herbelin).
```

Reply:
Wouldn't you prefer parentheses, as in:
 :cmd:`Fail` no longer catches anomalies (a behavior present since
       Coq version 8.11). :cmd:`Fail` now only catches user errors
  (`#15366 <https://github.com/coq/coq/pull/15366>`_,
  by Hugo Herbelin).